tests/alley/wp/alleyvate/features/test-clean-admin-bar.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class file for Test_Clean_Admin_Bar
+ *
+ * (c) Alley <info@alley.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @package wp-alleyvate
+ */
+
+namespace Alley\WP\Alleyvate\Features;
+
+use Alley\WP\Alleyvate\Feature;
+use Mantle\Testkit\Test_Case;
+
+/**
+ * Tests for the cleaning of the admin bar.
+ */
+final class Test_Clean_Admin_Bar extends Test_Case {
+	use \Mantle\Testing\Concerns\Refresh_Database;
+
+	/**
+	 * Feature instance.
+	 *
+	 * @var Feature
+	 */
+	private Feature $feature;
+
+	/**
+	 * Test admin bar cleaning.
+	 */
+	public function test_clean_admin_bar() { // // phpcs:ignore Generic.NamingConventions.ConstructorName.OldStyle
```

Comment:
This can be fixed by naming the function something different than the class:
```suggestion
	public function test_remove_admin_bar_nodes()
```